pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -224,11 +302,21 @@ func getHugepagesAllocationUnitOptions(hugepagesSize string, hugepagesCount int3
 	}
 }
 
-func addFile(ignitionConfig *igntypes.Config, src string, dst string, mode *int) error {
-	content, err := ioutil.ReadFile(src)
-	if err != nil {
-		return err
+func getRpsUnitOptions(rpsMask string) []*unit.UnitOption {
```

Comment:
should this service start before the kubelet service?

Reply:
The server is only triggered by the udev rule

Reply:
(Is not connected to the kubelet at all)

Reply:
I thought it should configure all masks before starting all containers?

Reply:
well, this is a new approach ending in the same way.
The udev will trigger an event for each new net device, so is enough to handle only the device.

The previous approach failed as it didn't take into account other net devices created after the container started.


This approach has 2 steps
1. Set the rps mask for each pod when the pod is started
2. Set the rps mask for each host net device as soon as is created

Reply:
Oh, isn't the service also started once on boot?

Reply:
No, the service is not started on boot, is started only by udev on net dev creation.